CMakeLists.txt

```diff
@@ -854,6 +855,8 @@ install(DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/ThirdParty/licenses" DESTINATION
 
 # OmniSciTypes.h local includes (for UDF/UDTF)
 install(FILES Shared/funcannotations.h DESTINATION "Shared/" COMPONENT "include")
+install(FILES Shared/InlineNullValues.h DESTINATION "Shared/" COMPONENT "include")
```

Comment:
Is it specific for DBEngine changes?

Reply:
nope, these are the header files that OmniSciTypes.h include.

Reply:
I'm just wondering should this go as standalone PR?

Reply:
Adding the missing header files as well as thrift files are simple bug fixes that are indirectly related to DBE packaging support (the omniscidb-feedstock would require these fixes anyway), and do not deserve a separate PR, IMHO.
If there is a particular need for the separation of these changesets then we can do it.

Reply:
No problem, I just worried a bit how omnisci is being built now without this changes :)

Reply:
Great then, thanks for clarification